app/controllers/users/confirmations_controller.rb

```diff
@@ -0,0 +1,28 @@
+class User::ConfirmationsController < Devise::ConfirmationsController
+  # GET /resource/confirmation/new
```

Comment:
Unless you are going to customize this controller, you don't need to have  the file at all.